src/Components/Publishing/Header/__test__/__snapshots__/FeatureHeader.test.tsx.snap

```diff
@@ -732,36 +714,36 @@ exports[`feature renders feature full header without a deck properly 1`] = `
   height: auto;
 }
 
-.c0[data-type="text"] .c8 {
+.c0[data-type="text"] .c9 {
```

Comment:
I just realized that snapshots would be far more useful if, like babel-plugin-styled-components, it printed out the component name. I wonder if that could be flipped on in tests?

Reply:
Looks like that was considered at one point: https://github.com/styled-components/jest-styled-components/issues/95